packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -382,6 +382,11 @@
     "@confirm_front_packaging_photo_button_label": {
         "description": "Button clicking on which confirms the picture of the front of product that user just took."
     },
+    "confirm_button_label": "Confirm",
+    "@confirm_button_label": {
+        "description": "Button clicking on which confirms the picture of the packaging that user just took."
```

Comment:
IMHO that's where the description makes no sense: its current value, and the fact that it exists.
We shouldn't have to explain the meaning of `"confirm_button_label"`, whose translation in English is "Confirm". It's just the label of a "Confirm" button.